targets/posix/nanoCLR/nanoCLR_native_posix.cpp

```
@@ -0,0 +1,205 @@
+
+uint16_t nanoCLR_GetNativeAssemblyCount()
+{
+    return (uint16_t)g_CLR_InteropAssembliesCount;
+}
+
+bool nanoCLR_GetNativeAssemblyInformation(const uint8_t *data, size_t size)
+{
+    if (data == nullptr)
+        return false;
+
+    const size_t requiredSize = g_CLR_InteropAssembliesCount * (5 * sizeof(uint16_t) + 128);
```

Clang (14.0.6)

[warning] variable 'requiredSize' is not initialized

(cppcoreguidelines-init-variables)